exporter/utils_test.go

```diff
@@ -29,18 +33,11 @@ func collect(c helpers.Collector) []prometheus.Metric {
 	return m
 }
 
-func filterMetrics(metrics []*helpers.Metric, filters []string) []*helpers.Metric {
+func filterMetrics(metrics []*helpers.Metric) []*helpers.Metric {
```

Comment:
Any reason to still have this function?

Reply:
I am going to rename it to zeroMetrics.
The idea is to collect all metrics, zero then and return a new list we can use to compare against a predefined list to ensure there are no missing metrics in the output. 
It will be useful when we improve the metrics renaming code.